src/sprout/Controllers/Admin/ManagedAdminController.php

```diff
@@ -400,14 +416,104 @@ public function _getExport()
         }
 
         // Apply filter
-        list($where, $params, $export->refine_fields) = $this->applyRefineFilter();
+        [$where, $params, $export->refine_fields] = $this->applyRefineFilter();
+        [$cols, $items] = $this->getRefinedDataPreview($where, $params, 'export');
 
+        // Create the itemlist for the preview section
+        if (count($items) == 0) {
+            $export->itemlist = '<p><i>No records found which match the refinebar clauses specified.</i></p>';
+
+        } else {
+            $itemlist = new Itemlist();
+            $itemlist->main_columns = $cols;
+            $itemlist->items = $items;
+            $export->itemlist = $itemlist->render();
+        }
+
+        return array(
+            'title' => 'Export for ' . Enc::html(strtolower($this->friendly_name)),
```

Comment:
I'm pretty sure title is encoded within the view, so you're double encoding here!

Reply:
I think you're right. It was already like this, I just moved it, but I'm gonna fix it!